go.mod

```diff
@@ -66,6 +66,11 @@ require (
 	sigs.k8s.io/yaml v1.4.0
 )
 
+require (
+	k8s.io/kubelet v0.29.1
+	k8s.io/kubernetes v1.29.1
```

Comment:
from which k8s versions are these supported? is the go pkg version correlated with the k8s release version?